src/entities/teams.js

```diff
@@ -0,0 +1,98 @@
+export class Team {
```

Comment:
Can we split Team and TeamApi into two different files maybe?

Reply:
It makes more sense having one entity per file, so I will follow your advice